controllers/pipeline/pipeline_adapter_test.go

```diff
@@ -18,10 +18,11 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/redhat-appstudio/integration-service/tekton"
 	"reflect"
 	"time"
 
+	"github.com/redhat-appstudio/integration-service/tekton"
+
```

Comment:
is this empty line on purpose?

Reply:
I think this is formatted by command `go fmt ./...` automatically.